templates/nw_dualaz_multitier_nat_with_eni.compound

```diff
@@ -121,6 +121,86 @@
             "AllowedPattern" : "(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})"
         }
     },
+    "Metadata" :
+    {
+        "AWS::CloudFormation::Interface" :
+        {
+            "ParameterGroups" :
+            [
+                {
+                    "Label" :
+                    {
+                        "default" : "VPC Configuration"
+                    },
+                    "Parameters" :
+                    [
+                        "CIDRVPC"
+                    ]
+                },
+                {
+                    "Label" :
+                    {
+                        "default": "Private Subnet Configuration"
```

Comment:
One space both before and after the colon, please. Looks like that applies to all files...